src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/LimitedPartnershipsApiApplication.java

```diff
@@ -5,6 +5,7 @@
 
 @SpringBootApplication
 public class LimitedPartnershipsApiApplication {
+    public static final String APP_NAMESPACE = "overseas-entities-api";
     public static void main(String[] args) {
         SpringApplication.run(LimitedPartnershipsApiApplication.class, args);
```

Comment:
You could add a `ApiLogger.debug` call in here saying "Starting Limited Partnerships API" or something to see that it works at run-time.

Reply:
Added this and get:
"created":"2024-10-25T14:59:58.934+01:00","event":"debug","namespace":"limited-partnerships-api","data":{"message":"Starting Limited Partnerships Api"}}

Reply:
👍 nice. But we should capitalise API in the message.

Reply:
Its the title of an app, so yes.

Reply:
shouldn't you have it at info level? debug is switched off in cidev onwards

Reply:
Well its just a stop gap for current test purposes, Sping Boot itself logs the start of the app.

Reply:
Spring already logs when the application has started. I guess this is just to prove the logging is working so I would say debug is fine.

Reply:
Happy for this to change to 'info' level. I still think API should be capitalised in the "Starting Limited Partnerships Api" output message.

Reply:
Capilatised API.